websites/Z/Zee5/presence.ts

```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+		clientId: "1103408220921397349",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+/**
+ * Get Timestamps
+ * @param {Number} videoTime Current video time seconds
+ * @param {Number} videoDuration Video duration seconds
+ */
+function getTimestamps(videoTime: number, videoDuration: number): number[] {
+	const startTime = Date.now();
+	return [
+		Math.floor(startTime / 1000),
+		Math.floor(startTime / 1000) - videoTime + videoDuration,
+	];
+}
+
+const getVideoStatus = (
+	presenceData: PresenceData,
+	video: HTMLVideoElement
+) => {
+	if (video.paused) {
+		presenceData.smallImageKey = "paused";
```

Comment:
Imgur links are used over Discord asset names in this repository. As you have many assets, you should put them in an enum like so:
https://github.com/PreMiD/Presences/blob/3aa7e3fb8e54999012de1d1528f37bc907ae4213/websites/C/Crunchyroll/presence.ts#L52-L58